insonmnia/dwh/sql.go

```diff
@@ -1609,27 +1612,38 @@ func (m *sqlStorage) newNetflagsWhere(builder squirrel.SelectBuilder, operator p
 
 func (m *sqlStorage) runQuery(conn queryConn, columns string, withCount bool, query string, args ...interface{}) (*sql.Rows, uint64, error) {
 	dataQuery := strings.Replace(query, "*", columns, 1)
+	var count uint64
```

Comment:
Querying for counts must be done before querying for data (because count rows should be closed before getting data rows).